### ISSUE TITLE
Expose export options from 3rd party tools, like Pandoc

### ISSUE THREAD

**Issue by HaoZeke** [typora/typora-issues#1451]
Given the fact that typora uses pandoc for pdf and other format exports, please expose pandoc settings, including TeX support.

**Comment by nnop** [comment]
+1

**Comment by Jerry-Chow** [comment]
I also want to customize my html format with pandoc
Hoping to have the customized setting on config file function

**Comment by Kiiv** [comment]
Hi, 

just want to upvote the possibility to choose the path of the pandoc executable manually. I'm working on a computer without admin right and can't install or change the path to add pandoc.

Thanks :)

**Comment by abnerlee** [comment]
https://support.typora.io/Export/#other-wiki-format

**Comment by platlas** [comment]
Not sure where is the right place to ask as there are several duplicates of this issue so sorry if this is not relevant here.

I still have problem with Pandoc detecting. I'm probably doing something wrong so I would like to ask if anybody can point me to the right direction.  

I installed Pandoc (2.14.1) via Homebrew on macos (Big Sur) and restarted Typora.
While trying to export epub, I got 'Require Pandoc to Continue' dialog. 
So I set Pandoc path in export -> general. Closed preferences window. And again: 'Require Pandoc to Continue' dialog. Opened preferences again and it looked like the path I had set was not there. So I've checked abnerworks.Typora.plist in Library/Preferences and pandocPath had correct value there. 
I have also tried to set Pandoc path directly to my .zshrc profile, but with no results.

Paths that I have used is: '/opt/homebrew/bin/pandoc' and also directly: '/opt/homebrew/Cellar/pandoc/2.14.1/bin/pandoc'.
